components/FormCollection/FormCollection.js

```diff
@@ -0,0 +1,138 @@
+import styled, { keyframes, css } from "styled-components";
+import RegularButton from "../Buttons/RegularButton";
+import ButtonWrapper from "../Buttons/ButtonWrapper";
+import FormInput from "../FormFlashcard/FormInput";
+
+export default function FormCollection({
+  headline,
+  actionMode,
+  changeActionMode,
+  currentCollection,
+  isFormClosing,
+  onEditCollection,
+}) {
+  async function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = Object.fromEntries(new FormData(event.target));
+    const { collectionName, collectionColor } = formData;
+
+    let updatedCollection = {
+      ...currentCollection,
+      title: collectionName,
+      color: collectionColor,
+    };
+
+    try {
+      await onEditCollection(updatedCollection);
+      event.target.reset();
+      changeActionMode("default");
+    } catch (error) {
+      console.error("Error while editing collection:", error);
+    }
+  }
+
+  return (
+    <StyledForm onSubmit={handleSubmit} $isFormClosing={isFormClosing}>
+      <StyledFormHeadline>{headline}</StyledFormHeadline>
+
+      <NewCollectionWrapper>
+        <CollectionNameWrapper>
+          <StyledLabel htmlFor="collectionName">Collection Name</StyledLabel>
+          <FormInput
+            name="collectionName"
+            maxlength="23"
```

Comment:
Please use camelcase. 
```suggestion
            maxLength="23"
```